### PR TITLE
Refs #23919 -- Removed usage of django.utils.decorators.ContextDecorator.

### PR DESCRIPTION
**Pull request by ChillarAnand**
ContextDecorator is available in standard library from Python3.2

### CODE REVIEW

File: django/utils/translation/__init__.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 """
 Internationalization support.
 """
+
```
Comment (charettes) [inline django/utils/translation/__init__.py]: Unnecessary whitespace change.

File: django/utils/decorators.py [hunk 2]
```diff
@@ -166,21 +161,6 @@ def callback(response):
...
-        A base class that enables a context manager to also be used as a decorator.
-        """
-        def __call__(self, func):
-            @wraps(func, assigned=available_attrs(func))
-            def inner(*args, **kwargs):
-                with self:
-                    return func(*args, **kwargs)
-            return inner
-
-
```
Comment (charettes) [inline django/utils/decorators.py]: This removal will have to go through a deprecation period.

File: django/utils/decorators.py [hunk 3]
```diff
@@ -1,10 +1,5 @@
 "Functions that help with dynamically creating decorators for views."
 
-try:
-    from contextlib import ContextDecorator
```
Comment (timgraham) [inline django/utils/decorators.py]: Leave 
```:
# For backwards-compatibility in Django 2.0
from contextlib import ContextDecorator  # noqa
```
